integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -42,51 +47,189 @@ def get_context_data(self, **kwargs):
+                ):
+                    pages_under_hix_threshold.append(hix_translation)
+
+            worst_page = (
+                pages_under_hix_threshold[0] if pages_under_hix_threshold else None
+            )
+
+            return {
+                "pages_under_hix_threshold": pages_under_hix_threshold,
+                "worst_page": worst_page if worst_page else None,
+                "hix_score": round(worst_page.hix_score, 2)
+                if worst_page is not None
+                else None,
+            }
+        return {}
+
+    def get_outdated_pages_context(self):
+        r"""
+        Extend context by info on outdated pages
+
+        :return: Dictionary containing the context for outdated pages
+        :rtype: dict
+        """
+        OUTDATED_THRESHOLD_DATE = datetime.now() - relativedelta(
+            days=settings.OUTDATED_THRESHOLD_DAYS
+        )
+
+        outdated_pages = (
+            PageTranslation.objects.filter(
+                last_updated__lte=OUTDATED_THRESHOLD_DATE.date(),
+                page__explicitly_archived=False,
+                language__slug=self.request.region.default_language.slug,
+            )
+            .order_by("page__id", "language__id", "-version", "last_updated")
+            .distinct("page__id", "language__id")
+            .all()
+        )
+        days_since_last_updated = (
+            (
+                datetime.today() - most_outdated_page.last_updated.replace(tzinfo=None)
+            ).days
+            if (most_outdated_page := outdated_pages[0] if outdated_pages else None)
+            else None
+        )
+
+        OUTDATED_THRESHOLD_DATE = OUTDATED_THRESHOLD_DATE.strftime("%Y-%m-%d")
+
+        return {
+            "outdated_pages": outdated_pages,
+            "most_outdated_page": most_outdated_page,
+            "days_since_last_updated": days_since_last_updated,
+            "outdated_threshold_date": OUTDATED_THRESHOLD_DATE,
+        }
+
+    def get_broken_links_context(self):
+        r"""
+        Extend context by info on broken links
+
+        :return: Dictionary containing the context for broken links
+        :rtype: dict
+        """
+        invalid_urls = filter_urls(self.request.region.slug, "invalid")[0]
+        invalid_url = invalid_urls[0] if invalid_urls else None
+
+        relevant_translation = (
+            invalid_url.region_links[0].content_object if invalid_url else None
+        )
+
+        return {
+            "broken_links": invalid_urls,
+            "relevant_translation": relevant_translation,
+        }
+
+    def get_automatically_saved_pages(self):
+        r"""
+        Extend context by info on automatically saved pages
+
+        :return: Dictionary containing the context for auto saved pages
+        :rtype: dict
+        """
+        automatically_saved_pages = (
+            PageTranslation.objects.filter(
+                status=status.AUTO_SAVE,
+                page__explicitly_archived=False,
```

Comment:
It would probably also make sense to exclude implicitely archived pages here.

Reply:
@PeterNerlich Here as well, you resolved the conversation, but the issue does not seem to be resolved?

Reply:
Because Johannes and I went over this today. The changes are just not pushed yet. We will continue work on this tomorrow morning.